src/test/java/com/gargoylesoftware/htmlunit/javascript/host/file/FileTest.java

```diff
@@ -136,6 +139,40 @@ public void typeHtmlUnit() throws Exception {
     public void typeEmpty() throws Exception {
         type("");
     }
+    
+    @Test
+    @Alerts({"myfilename", "application/octet-stream", "1000", "0"})
+    @NotYetImplemented(IE)
+    public void fileConstructor() throws Exception {
+
+		final String html = "<html><head><script>\n" + "	function test() {\n"
+				+ "		var newFile = new File([], 'myfilename', {type: 'application/octet-stream', lastModified: '1000'});"
+				+ "     alert(newFile.name);"
+				+ "     alert(newFile.type);"
+				+ "     alert(newFile.lastModified);"
+				+ "     alert(newFile.size);"
+				+ " } "
+				+ "</script></head><body onload='test()'>\n"
+				+ "</body>\n" + "</html>";
+		loadPageWithAlerts2(html);
+	}
+
+    @Test
+    @Alerts({"myfilename", "", "true", "0"})
+    @NotYetImplemented(IE)
```

Comment:
This one errors right now but is ignored. It still logs some errors though. Is there an easy way to not even run those tests without having to add a FileTest.properties?